crypto/evp/encode.c

```diff
@@ -215,13 +265,17 @@ int EVP_EncodeBlock(unsigned char *t, const unsigned char *f, int dlen)
     return ret;
 }
 
+int EVP_EncodeBlock(unsigned char *t, const unsigned char *f, int dlen)
+{
+    return evp_encodeblock_int(NULL, t, f, dlen);
+}
+
 void EVP_DecodeInit(EVP_ENCODE_CTX *ctx)
 {
     /* Only ctx->num is used during decoding. */
     ctx->num = 0;
     ctx->length = 0;
     ctx->line_num = 0;
-    ctx->expect_nl = 0;
```

Comment:
So flags don't need initialization? If so how come? I mean if you rely on structure being memset to 0, then you wouldn't need any of initializations. But if you zero them explicitly, then flags deserve it as much, don't they?

Reply:
The structure is zalloc'd so none of these need initialisation. I removed them all.